sdk/python/feast/client.py

```diff
@@ -716,22 +729,25 @@ def ingest(
 
         # Update the feature set based on PyArrow table of first row group
         if force_update:
-            feature_set.infer_fields_from_pa(
-                table=pq_file.read_row_group(0),
-                discard_unused_fields=True,
-                replace_existing_features=True,
-            )
-            self.apply(feature_set)
+            if isinstance(feature_set, FeatureSet):
```

Comment:
I think we can probably safely remove force_update here. Its better if people run that as a separate step. Not sure if anyone is using it right now internally, and I think as written it was partly broken any ways.

Reply:
I think we can raise this as a new Issue? so the linter error will be separate case

Reply:
Ok, can you do that please.